src/main/scala/cromwell/binding/IOInterface.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding
```

Comment:
Well then I guess it depends on do we consider that any workflow language would need basic IO capabilities (read a file, write a file, list files in a directory...), or not. It's not obvious I don't know if CWL would actually have the need for this in the language itself.